src/config/config.ts

```diff
@@ -53,6 +53,12 @@ export class RaftConfig {
     if (tokenTicker === 'wstETH-v1') {
       return 'wstETH';
     }
+    if (tokenTicker === 'rETH-v1') {
+      return 'rETH';
+    }
+    if (tokenTicker === 'wcrETH-v1') {
```

Comment:
I think we don't need to prefix rETH since it wasn't used as collateral for v1. Similar thing with wcrETH, it was used only for v1. It's minor, can be changed after.